cmake/modules/hunter_check_toolchain_for_define.cmake

```diff
@@ -0,0 +1,29 @@
+# Copyright (c) 2016 NeroBurner
+# All rights reserved.
+
+include(hunter_test_string_not_empty)
+
+function(hunter_check_toolchain_for_define definition output_var)
+  # the toolchain path must be set
+  hunter_test_string_not_empty("${HUNTER_TOOLCHAIN_ID_PATH}")
+  set(TOOLCHAIN_INFO_FILE "${HUNTER_TOOLCHAIN_ID_PATH}/toolchain.info")
+
+  # read toolchain.info file into memory, each line a entry in a list
+  file(STRINGS "${TOOLCHAIN_INFO_FILE}" toolchain_contents)
+
+  # clean ouptut variable
+  set(definition_found)
+
+  foreach(line ${toolchain_contents})
+    # find the specified define
+    string(REGEX MATCH "^#define ${definition} .*$" match "${line}")
+    if(match)
+      # extract the definition
+      string(REGEX REPLACE "^#define ${definition} " "" definition_found "${match}")
```

Comment:
The name of variable `definition_found` doesn't match the meaning. I guess the value of variable `definition_found` should be `YES` or `NO`.